src/lib/tasks/server/execute_server_prepared_statement.hpp

```diff
@@ -4,12 +4,14 @@
 
 namespace opossum {
 
-class ExecuteServerPreparedStatement : public ServerTask {
+class SQLQueryPlan;
```

Comment:
why do we do this compared to just including the respective headers?

Reply:
Something I've learnt in various 3D Point Cloud projects -- systematic forward declarations can reduce compile time significantly by not having to include unnecessary headers